aptos-move/framework/aptos-token/doc/token.md

```diff
@@ -108,6 +108,7 @@ Checkout our developer doc on our token standard https://aptos.dev/concepts/coin
 -  [Function `direct_deposit`](#0x3_token_direct_deposit)
 -  [Function `assert_collection_exists`](#0x3_token_assert_collection_exists)
 -  [Function `assert_tokendata_exists`](#0x3_token_assert_tokendata_exists)
+-  [Function `assert_not_standard_reserved_property`](#0x3_token_assert_not_standard_reserved_property)
```

Comment:
s/not/non?

Reply:
standard or framework? We should make it consistent.

Reply:
will use `standard` in code for consistency

Reply:
you change the naming as `non_xx` at other places but not here?

Reply:
this file is auto-generated from the code.